src/SFA.DAS.AANHub.Api/Models/CreatePartnerModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SFA.DAS.AANHub.Application.Partners;
+
+namespace SFA.DAS.AANHub.Api.Models
+{
+    public class CreatePartnerModel
+    {
+        public string UserName { get; set; } = null!;
+        public string Email { get; set; } = null!;
+        public string Name { get; set; } = null!;
+        public DateTime Joined { get; set; }
+        public List<int>? Regions { get; set; }
+        public string? Information { get; set; }
+        public string Organisation { get; set; } = null!;
+
+        public static implicit operator CreatePartnerMemberCommand(CreatePartnerModel model) => new()
```

Comment:
Coverage missing for the operator

Reply:
It's covered as part of `CreatePartnersMemberCommandHandlerTests`, isn't it?

Reply:
I expect to see explicit test, with assertions on each field.

Reply:
Added here now: https://github.com/SkillsFundingAgency/das-aan-hub-api/blob/7d8c7338247663f8460af149a795a511a4610dd1/src/SFA.DAS.AANHub.Api.UnitTests/Controllers/PartnersControllerTests.cs#L79